apps/api/src/app/subscribers/usecases/create-subscriber/create-subscriber.usecase.ts

```diff
@@ -41,4 +56,17 @@ export class CreateSubscriber {
 
     return subscriber;
   }
+
+  @CachedEntity({
+    builder: KeyGenerator.subscriber,
+  })
+  private async fetchSubscriber({
+    subscriberId,
+    _environmentId,
+  }: {
+    subscriberId: string;
+    _environmentId: string;
+  }): Promise<SubscriberEntity | null> {
```

Comment:
Can we enforce the param, passed to the function to make sure that it's enforce by TS? That hsubscriberId and environment id is indeed passed?

Reply:
Tried it about a week ago and a bit struggled to do so.